### PR TITLE
added function to check a string array contains a string

### PR DESCRIPTION
**Pull request by IncredibleMrTim**
### What

Added additional helper function to check if a string array contains a given string

### How to review

Pull the code and run `go test helper/string_array_contains_test.go`

### Who can review

Anyone

### CODE REVIEW

File: assets/templates/partials/header.tmpl [hunk 1]
```diff
@@ -244,7 +244,7 @@
             </ul>
         </nav>
     </div>
-    {{if not .SearchDisabled}}
+    <!--{{if not .SearchDisabled}}
```
Comment (s-xton) [inline assets/templates/partials/header.tmpl:247]: typo ?

File: helper/string_array_contains.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package helper
+
```
Comment (s-xton) [inline helper/string_array_contains.go:2]: comparator ?

File: helper/string_array_contains_test.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+func TestStringArrayContains(t *testing.T) {
+	Convey("That true is returned if the array contains the given comparitor", t, func() {
+		stringArray := []string{"cat", "dog"}
+		got := helper.StringArrayContains("cat", stringArray)
+		want := true
+		So(got, ShouldEqual, want)
+	})
+
```
Comment (s-xton) [inline helper/string_array_contains_test.go:17]: typo on congain

**Review by s-xton — CHANGES_REQUESTED** [review]
couple of minor comments

File: assets/templates/partials/header.tmpl (hunk 1 above)
Comment (IncredibleMrTim) [reply assets/templates/partials/header.tmpl:247]: fixed this, although for now i am leaving the commented code in there as i may still use it

File: helper/string_array_contains_test.go (hunk 3 above)
Comment (IncredibleMrTim) [reply helper/string_array_contains_test.go:17]: fixed

File: helper/string_array_contains.go (hunk 2 above)
Comment (IncredibleMrTim) [reply helper/string_array_contains.go:2]: fixed

**Review by s-xton — APPROVED** [review]

**Pull request merged**